src/applications/letters/utils/helpers.jsx

```diff
@@ -95,24 +95,6 @@ const commissaryLetterContent = (
   </div>
 );
 
-// Benefit Summary Letter Help Instructions
-export const bslHelpInstructions = (
-  <div>
-    <p>
-      If your service period or disability status information is incorrect,
-      please send us a message through VA’s{' '}
-      <a
-        target="_blank"
-        rel="noopener noreferrer"
-        href="https://iris.custhelp.com/app/ask"
-      >
-        Inquiry Routing & Information System (IRIS)
-      </a>
-      . VA will respond within 5 business days.
-    </p>
-  </div>
-);
-
```

Comment:
This is the only one I'm concerned about, referenced [here](https://github.com/department-of-veterans-affairs/vets-website/blob/7e40a773626d8f0f58742fed6c90f9586707db6c/src/applications/letters/containers/LetterList.jsx#L27).  But it looks like we always reference the [help desk](https://github.com/department-of-veterans-affairs/vets-website/blob/7e40a773626d8f0f58742fed6c90f9586707db6c/src/applications/letters/containers/LetterList.jsx#L155) at the bottom of the app, so I think we're covered.